src/Maestro/Maestro.ContainerApp/Api/Controllers/AssetsControllers.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+using Maestro.ContainerApp.Api.Models;
+using Maestro.ContainerApp.Utils;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.DarcLib;
+
+namespace Maestro.ContainerApp.Api.Controllers;
+
+/// <summary>
+///   Exposes methods to Read/Query <see cref="Asset"/>s and modify <see cref="AssetLocation"/> information
```

Comment:
I guess this controller was originally supposed to do something very different?

Reply:
That's quite possible. If we ever productionize these changes, we'll need to review the comments.